Assets/MixedRealityToolkit/_Core/Utilities/Lines/DataProviders/SimpleLineDataProvider.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.DataProviders
+{
+    /// <summary>
+    /// A simple line with two points.
+    /// </summary>
+    public class SimpleLineDataProvider : BaseMixedRealityLineDataProvider
+    {
+        [SerializeField]
```

Comment:
tooltip

Reply:
Not all serialized fields are shown in the inspector, and as such, don't need tooltips.